sponsor.html

```diff
@@ -394,11 +513,54 @@ <h2> Sponsorship Prospectus </h2>
     <section class="footer">
       Copyright &copy; 2019 - 2020 Technica. All Rights Reserved.
```

Comment:
For most sites I've seen, the current year is used. Using a prior year opens us up to copyright issues, unfortunately. Can we change this so it's just `2020 Technica`?